src/components/Differential/DifferentialSearch.jsx

```diff
@@ -435,6 +435,7 @@ class DifferentialSearch extends Component {
       multisetFiltersVisibleDifferential: false,
       sigValP: this.state.uSettingsP.defaultSigValueP,
       selectedColP: [],
+      selectedOperatorP: [this.state.uSettingsP.defaultselectedOperatorP],
```

Comment:
This is the only line that needed to be changed...the rest of the diff is just formatting